srcs/server/src/user/user.service.ts

```diff
@@ -35,6 +35,10 @@ export class UserService {
     achievements: true,
   };
 
+  readonly includedMatchRelations: object = {
+    players: true,
+  };
+
   readonly userStatus = Object.freeze({
     AWAY: 0,
```

Comment:
when do you set the userStatus ? 

And how do you know that a user is away ?

Reply:
Oh you are right, I this this is obsolete and should be removed. I will check tomorrow and repush on the same PR if needed.

Reply:
Done